easybuild/easyconfigs/m/msprime/msprime-1.2.0-foss-2021b.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'PythonBundle'
+
+name = 'msprime'
+version = '1.2.0'
+
+homepage = 'https://msprime.readthedocs.io'
+description = "msprime is a coalescent simulator and library for processing tree-based genetic data."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),  # for numpy
+    ('GSL', '2.7'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('svgwrite', '1.4.3', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'checksums': ['a8fbdfd4443302a6619a7f76bc937fc683daf2628d9b737c891ec08b8ce524c3'],
+    }),
+    ('tskit', '0.5.4', {
+        'checksums': ['be1a6381c72a95f0011bea26ca317e1b4503b8d2dcd87096c68aa16c8946c6fe'],
+    }),
+    ('ruamel.yaml.clib', '0.2.7', {
+        'modulename': False,
+        'checksums': ['1f08fd5a2bea9c4180db71678e850b995d2a5f4537be0e94557668cf0f5f9497'],
+    }),
+    ('ruamel.yaml', '0.17.21', {
+        'checksums': ['8b7ce697a2f212752a35c1ac414471dc16c424c9573be4926b56ff3f5d23b7af'],
+    }),
+    ('demes', '0.2.2', {
+        'checksums': ['8eebe4b236b2d19e057228892f5b5f53f3e0d127ca9128efead1728a65115dac'],
+    }),
+    ('newick', '1.6.0', {
+        'checksums': ['5610335826c0afd35afe66ebbacba0881e69f25097418c2f08456be629be6eac'],
+    }),
+    (name, version, {
+        'checksums': ['60025ad9fd30d827a72ae6e760b6cff07a1d0e169b2c1da1024cb0fc23e4a7aa'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/msp', 'bin/mspms', 'bin/tskit', 'bin/demes'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```

Comment:
@akesandgren don't suppose you feel like adding a `sanity_check_command`?

Reply:
can do...

Reply:
And done.